open-ce/build_feedstock.py

```diff
@@ -103,6 +103,8 @@ def build_feedstock_from_command(command, # pylint: disable=too-many-arguments,
     Build a feedstock from a build_command object.
     '''
     utils.check_if_conda_build_exists()
+    utils.check_cuda_version_match()
```

Comment:
These should only be called if the `build_type` is cuda.

Reply:
Makes sense; Now only invokes if `cuda` was specified in build type.